### DISCUSSION TITLE
Is there a guide for IPv6 on Podman?

### DISCUSSION THREAD

**Discussion by vedranmiletic** [docker-mailserver/docker-mailserver#2473] [Q&A]
There is a [guide for enabling IPv6 on Docker](https://docker-mailserver.github.io/docker-mailserver/edge/config/advanced/ipv6/) and there is a [guide for running via Podman](https://docker-mailserver.github.io/docker-mailserver/edge/config/advanced/podman/). Is there a way to enable IPv6 when running via Podman?

FWIW, it doesn't "just work", at least not for me. I'm using Arch Linux (btw) and Podman 4.0.0.

**Comment by vedranmiletic** [comment]
I have been trying to get this working over the last couple of days. I'll list what I have tried and isn't working:

- https://docker-mailserver.github.io/docker-mailserver/edge/config/advanced/ipv6/ (IPv6 is disabled regardless of the settings)
- https://blog.vcambria.org/exploit-ipv6-using-podman/ (the network isn't used by `podman-compose`)
- `slirp4netns` (unsupported by `podman-compose`)

**Comment by georglauterbach** [comment]
Our docs do not currently have a dedicated Podman + IPv6 guide. If you get it to work, we'd very much appreciate if you add your findings to our docs so others can profit from it as well :)

**Comment by vedranmiletic** [comment]
To paraphrase Edison, I’ve successfully found many ways that will not work, but not yet the one that will work.
